components/spin/index.tsx

```diff
@@ -126,6 +130,7 @@ const Spin: React.FC<SpinClassProps> = (props) => {
       [`${prefixCls}-lg`]: size === 'large',
       [`${prefixCls}-spinning`]: spinning,
       [`${prefixCls}-show-text`]: !!tip,
+      [`${prefixCls}-fullscreen`]: !!fullscreen,
```

Comment:
```suggestion
      [`${prefixCls}-fullscreen`]: fullscreen,
```

Reply:
it's already boolean hahaha :man_facepalming: